src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -712,14 +686,15 @@ private void moveUnitsToDefendTerritories(final Map<Territory, ProTerritory> mov
           }
           if (moveMap.get(t).getBattleResult() == null) {
             moveMap.get(t).setBattleResult(
-                battleUtils.estimateDefendBattleResults(player, t, moveMap.get(t).getMaxEnemyUnits(), defendingUnits,
-                    moveMap.get(t).getMaxEnemyBombardUnits()));
+                ProBattleUtils.estimateDefendBattleResults(player, t, moveMap.get(t).getMaxEnemyUnits(),
+                    defendingUnits, moveMap.get(t).getMaxEnemyBombardUnits()));
           }
           final ProBattleResult result = moveMap.get(t).getBattleResult();
           final boolean hasFactory = ProMatches.territoryHasInfraFactoryAndIsLand(player).match(t);
           if (result.getWinPercentage() > maxWinPercentage
-              && ((t.equals(myCapital) && result.getWinPercentage() > (100 - WIN_PERCENTAGE))
-                  || (hasFactory && result.getWinPercentage() > (100 - MIN_WIN_PERCENTAGE)) || result.getTUVSwing() >= 0)) {
+              && ((t.equals(myCapital) && result.getWinPercentage() > (100 - ProData.winPercentage))
```

Comment:
The computation of this if statement could  be moved to `ProBattleResult`, would allow testing and for this client code to be much simpler.